.gitignore

```
@@ -14,3 +14,6 @@ out/
 
 # Visual Studio Code Extension package
 *.vsix
+
+coverage
+\.nyc_output
```

```suggestion
.nyc_output
```

Why is this escaped?

---

Good question, removed.